locust/core.py

```diff
@@ -350,6 +350,13 @@ def client(self):
         """
         Reference to the :py:attr:`client <locust.core.Locust.client>` attribute of the root 
         Locust instance.
-        """
+        """ 
         return self.locust.client
 
+    @property
+    def read_configuration(self):
```

Comment:
how about just `configuration`? because it defined as a property instead of a function